charts/ibm-cpd-wkc-instance/values.yaml

```diff
@@ -0,0 +1,7 @@
+name: wkc-cr
+cpd_namespace: gitops-cp4d-instance
+license: Enterprise
+version: 4.0.5
+storageVendor: portworx
+wkc_set_kernel_params: True
```

Comment:
Is this value supposed to be used in the `setkernelparams.yaml` template to conditionally apply the value? I don't see it used anywhere. Also, the value is a boolean so it should be `true`

Reply:
no, this is required in CR as per documentation : https://www.ibm.com/docs/en/cloud-paks/cp-data/4.0?topic=catalog-installing-watson-knowledge-service